tests/compat.sh

```diff
@@ -534,6 +534,15 @@ add_mbedtls_ciphersuites()
     esac
 }
 
```

Comment:
I know it wasn't a requirement of #1785, but since `compat.sh` is now skipping static ECDH cipher suites automatically, I think we should remove the explicit exclusions in `.travis.yml` and `all.sh`.

Reply:
No explicit exclusions found in `all.sh`, so I only removed that in `.travis.yml`.

Reply:
I also found this doesn't work for a more recent OpenSSL (e.g 1.1.1f on Travis CI).
https://github.com/Mbed-TLS/mbedtls/blob/ffb92b078952449c12c9f1ea0fde6016ba32f45c/tests/compat.sh#L823-L824

Reply:
Actually, the comment (which was added later by a different person) is wrong. `Cipher is (NONE)` indicates a null cipher, not a cipher that isn't supported, and that's still true at least in OpenSSL 1.1.1. So what this piece of code really does is to automatically mark null cipher suites as skipped if OpenSSL doesn't support them.

I don't think we need to change the code: it's working well enough. But before we forget, please fix the comment.

Reply:
OK, I see. But `Cipher is (NONE)` is also printed if there is a invalid ciphersuite in old OpenSSL. So I misunderstood the code here. Let me update the comment.